argo-lite/package.json

```diff
@@ -14,6 +14,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.11.2",
     "@fortawesome/react-fontawesome": "^0.1.7",
     "@svgr/webpack": "4.3.2",
+    "@trendmicro/react-sidenav": "^0.5.0",
```

Comment:
are we using this?

Reply:
@aupadhayay3 Is this used? I think we can remove this.

Reply:
You can remove this

Reply:
@aupadhayay3 You can make a commit to remove this. Btw there's no need to close this PR. When you make more commits to this branch I will receive updates and will be able to continue the code review in the same PR.